day_two_guide/topics/alerts_using_prometheus.adoc

```diff
@@ -0,0 +1,33 @@
+////
+Creating alerts using Prometheus
+
+Module included in the following assemblies:
+
+* day_two_guide/environment_health_checks.adoc
+////
+
+While the topics in this section are for manually checking the health of an
+{product-title} component, you can integrate {product-title} with Prometheus to
+create visuals and alerts to help diagnose any environment issues before they
+arise. These issues can include if a node goes down, if a pod is consuming too
+much CPU or memory, and more.
+
+See the
```

Comment:
If that topic were in modules, this would be a great place for an include instead of an xref.